resources/views/components/subscribe-form/index.blade.php

```diff
@@ -0,0 +1,26 @@
+@props(['product_id'])
+@slots(['subscribe', 'login'])
+
+<graphql-mutation
+    v-if="user?.email"
+    v-cloak
+    query="@include('rapidez-product-alert::graphql.product-alert-subscribe')"
+    :variables='{product_id: {{ $product_id }}, email: user.email}'
+    :callback="() => { window.app.$emit('alerts-updated') }"
+    :notify="{message: '@lang('We will let you know once this product is back in stock')', type: 'success'}"
+>
+    <div slot-scope="{ mutate }">
+        @if($subscribe->isNotEmpty())
+            {{ $subscribe }}
+        @else
+            @include('rapidez-product-alert::partials.subscribe')
+        @endif
```

Comment:
Would be nice to have a directive for this like;
```
@slot('subscribe')
    @include('rapidez-product-alert::partials.subscribe')
@endslot
```
And added within: https://github.com/rapidez/blade-directives, thoughts @indykoning?

Reply:
Implementation: https://github.com/rapidez/blade-directives/pull/12